components/measurement/nettests/telegram.js

```diff
@@ -67,13 +50,16 @@ const TelegramDetails = ({ measurement, render }) => {
 
   if (!telegramWebOK || !telegramDesktopOK) {
     anomaly = true
-    hint = 'Telegram is not working'
+    hint = <FormattedMessage id='Measurement.Status.Hint.Telegram.Blocked' />
   }
 
   return (
     render({
       status: anomaly ? 'anomaly': 'reachable',
```

Comment:
Is this not being translated? Or are these used as keys to affect how to present some views in other pages?

If the later maybe defining some named constants is more appropriate.

Reply:
Assuming you meant _Is this not being used?_:
I wanted to return to this after I restructure the SummaryText component. I will update it accordingly now.